### ISSUE TITLE
Issue with track name 'NoneType not subscriptable'

### ISSUE THREAD

**Issue by HStep20** [rnagabhyrava/plex-playlist-sync#33]
Whenever I try to run the script on this specific playlist (https://open.spotify.com/playlist/0Nq1trkWUxqlG1QjNnH5gw) it crashes with the error 'NoneType object is not subscriptable'. Since the logging isn't very robust, Ill try to spin up a dev environment tomorrow and see exactly what song is causing the issue, if you aren't able to get to it

Error is as follows:

```
today at 11:21:21 PMTraceback (most recent call last):
today at 11:21:21 PM  File "/app/./plex-playlist-sync/run.py", line 71, in <module>
today at 11:21:21 PM    spotify_playlist_sync(sp, plex, userInputs)
today at 11:21:21 PM  File "/app/plex-playlist-sync/utils/spotify.py", line 92, in spotify_playlist_sync
today at 11:21:21 PM    tracks = _get_sp_tracks_from_playlist(
today at 11:21:21 PM  File "/app/plex-playlist-sync/utils/spotify.py", line 74, in _get_sp_tracks_from_playlist
today at 11:21:21 PM    list(map(extract_sp_track_metadata, sp_playlist_tracks["items"]))
today at 11:21:21 PM  File "/app/plex-playlist-sync/utils/spotify.py", line 58, in extract_sp_track_metadata
today at 11:21:21 PM    title = track["track"]["name"]
today at 11:21:21 PMTypeError: 'NoneType' object is not subscriptable
```
An expected behavior could be to run through a playlist, and perhaps log an error of the song, instead of crashing completely.

**Comment by rnagabhyrava** [comment]
From spotify UI there are 352 tracks. But API returned 353 tracks it somehow added an empty track with all None fields between 351 and 352 and that caused the error. This could be an issue with upstream spotipy package.

Fixed in #34 pull latest image for fix. Let me know how it goes.

**Comment by HStep20** [comment]
Looks like that fixed it, Thanks for the quick reply!
